x-pack/plugins/monitoring/common/http_api/logstash/post_logstash_cluster_pipelines.ts

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import * as rt from 'io-ts';
+import { clusterUuidRT, ccsRT, timeRangeRT, paginationRT, sortingRT } from '../shared';
+
+export const postLogstashClusterPipelinesRequestParamsRT = rt.type({
+  clusterUuid: clusterUuidRT,
+});
+
+export const postLogstashClusterPipelinesRequestPayloadRT = rt.intersection([
+  rt.partial({
+    ccs: ccsRT,
+    sort: sortingRT,
+    queryText: rt.string,
```

Comment:
The original type had a default value of `''` when `queryText` was not sent. Should we have this validation here?